chain/chain/src/test_utils.rs

```diff
@@ -1468,10 +1468,16 @@ mod test {
         let prod_duration = start.elapsed();
         assert_eq!(naive_result, prod_result);
         // production implementation is at least 50% faster
-        assert!(2 * naive_duration > 3 * prod_duration);
+        assert!(
+            2 * naive_duration > 3 * prod_duration,
+            "naive duration vs production {:?} {:?}",
+            naive_duration,
+            prod_duration
+        );
     }
 
     #[test]
+    #[ignore]
```

Comment:
add a comment with github issue that explains more info on why we ignored this test.

Reply:
https://github.com/near/nearcore/issues/5836